### PR TITLE
Mark build_test builds as bringup.

### PR DESCRIPTION
**Pull request by godofredoc**
They require an additional dimension to be passed as part of the .ci.yaml configuration.

Bug: https://github.com/flutter/flutter/issues/125897

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I signed the [CLA].
- [X] I listed at least one issue that this PR fixes in the description above.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making, or this PR is [test-exempt].
- [X] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by keyonghan** [comment]
Instead of marking them as flaky, can we simply add the `os` dimension to green the tree?

File: .ci.yaml [hunk 1]
```diff
@@ -2139,6 +2139,7 @@ targets:
 
   - name: Linux_build_test flutter_gallery__transition_perf
     recipe: devicelab/devicelab_drone_build_test
+    bringup: true # https://github.com/flutter/flutter/issues/1258979
```
Comment (drewroengoogle) [inline .ci.yaml:2142]: Nit: This issue is returning a 404, is it supposed to be https://github.com/flutter/flutter/issues/125897 ?

**Review by keyonghan — APPROVED** [review]
LGTM

File: .ci.yaml (hunk 1 above)
Comment (godofredoc) [reply .ci.yaml]: Actually did both, after it starts passing I'll remove the bringup.

**Pull request merged**